data/seeds/seed-database.sql

```diff
@@ -0,0 +1,57 @@
+-- Connect to the database using the psql client:
+--
+--   $ psql postgresql://localhost/bonuscalc
+--
+-- Use the correct url for your database and then run this file:
+--
+--   bonuscalc=# \i seed-database.sql
+--
+-- The API can then be started using `dotnet run`
+
+BEGIN;
+
+-- Remove any existing data
+TRUNCATE TABLE
```

Comment:
Is it risky to have this here when the user could be connected to the production DB via a ssh session and accidently wipe production?

Reply:
The migrations leave a record in the `trades` table so the script will fail on a newly setup database without them - maybe a sanity check at the start that tries to detect production and exits?

Reply:
I think that would be wise